src/suparnaNJune20/Assignment_24/PhoneNoValidation.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Assignment 24 :- 5th Jul'2020
+Validate given phone number.
+
++91 9765463742 --> valid
+09765463742 --> valid
+976 546 3742 --> valid
+9765463742 --> valid
+
+9765463A42 --> Invalid
+976546342 --> Invalid
+6976546342 --> Invalid
+*/
+package suparnaNJune20.Assignment_24;
+
+public class PhoneNoValidation {
+void validPhoneNo(String phoneNo) {
+		switch (phoneNo.length()) {
+		case 14:
+			if (phoneNo.charAt(0) == '+' && phoneNo.charAt(1) == '9' && phoneNo.charAt(2) == '1'
+					&& phoneNo.charAt(3) == ' '
+					&& (phoneNo.charAt(4) == '9' || phoneNo.charAt(4) == '8' || phoneNo.charAt(4) == '7')
+					&& (Character.isDigit(phoneNo.charAt(5))) && (Character.isDigit(phoneNo.charAt(6)))
+					&& (Character.isDigit(phoneNo.charAt(7))) && (Character.isDigit(phoneNo.charAt(8)))
+					&& (Character.isDigit(phoneNo.charAt(9))) && (Character.isDigit(phoneNo.charAt(10)))
+					&& (Character.isDigit(phoneNo.charAt(11))) && (Character.isDigit(phoneNo.charAt(12)))
+					&& (Character.isDigit(phoneNo.charAt(13))))
+				System.out.println(phoneNo + " is valid phone no");
+			else
+				System.out.println(phoneNo + " is in valid phone no");
+
+			break;
+		case 11:
+			if (phoneNo.charAt(0) == '0'
+					&& (phoneNo.charAt(1) == '9' || phoneNo.charAt(1) == '8' || phoneNo.charAt(1) == '7')
+					&& (Character.isDigit(phoneNo.charAt(2))) && (Character.isDigit(phoneNo.charAt(3)))
+					&& (Character.isDigit(phoneNo.charAt(4))) && (Character.isDigit(phoneNo.charAt(5)))
+					&& (Character.isDigit(phoneNo.charAt(6))) && (Character.isDigit(phoneNo.charAt(7)))
+					&& (Character.isDigit(phoneNo.charAt(8))) && (Character.isDigit(phoneNo.charAt(9)))
+					&& (Character.isDigit(phoneNo.charAt(10))))
```

Comment:
Same applied here.

Reply:
Replaced with function which check string char is digit or not .